tests/test_flows.py

```diff
@@ -44,14 +44,17 @@ def start_test(self, test: str):
         status(test + " starting")
 
     def check_log(self, entity_or_radio: Union[env.ALEntity, env.Radio], regex: str,
-                  start_line: int = 0) -> bool:
-        '''Verify that the logfile for "entity_or_radio" matches "regex", fail if not.'''
-        return self.wait_for_log(entity_or_radio, regex, start_line, 0.3)
+                  start_line: int = 0, fail_on_mistmatch: bool = True) -> bool:
```

Comment:
Changes in test_flows should be done in another commit.

Reply:
fixed.